rising/transforms/compose.py

```diff
@@ -1,24 +1,31 @@
-from typing import Sequence, Union
+from typing import Sequence, Union, Callable, Any
 from rising.utils import check_scalar
-from .abstract import AbstractTransform, RandomProcess
+from rising.transforms import AbstractTransform, RandomProcess
+from rising.loading import default_transform_call
+
 
 __all__ = ["Compose", "DropoutCompose"]
 
 
 class Compose(AbstractTransform):
-    def __init__(self, *transforms):
+    def __init__(self, *transforms,
+                 transform_call: Callable[[Any, Callable], Any] = default_transform_call):
         """
         Compose multiple transforms
 
         Parameters
         ----------
         transforms: Union[AbstractTransform, Sequence[AbstractTransform]]
             one or multiple transformations which are applied in consecutive order
+        transform_call: Callable[[Any, Callable], Any], optional
+            function which determines how transforms are called. By default
+            Mappings and Sequences are unpacked during the transform.
         """
         super().__init__(grad=True)
         if isinstance(transforms[0], Sequence):
             transforms = transforms[0]
         self.transforms = transforms
+        self.transform_call = transform_call
```

Comment:
I don't know, if this is really a good idea. This leads to multiple type-checks and might also be an additional source of errors. I would rather leave it to the user to ensure, that the transform chain is working as expected.

Reply:
I think the overhead which is introduced by the type checks is extremely small but just to be sure, I changed the default to the normal dict call (this only introduces an additional function call).